test/models/organization_test.rb

```diff
@@ -3,7 +3,37 @@
 require 'katello_test_helper'
 
 module Katello
-  class OrganizationTestDelete < ActiveSupport::TestCase
+  class OrganizationTest < ActiveSupport::TestCase
+    def test_org_valid_labels
+      assert Organization.new(:name => 'name', :label => 'org').valid?
+      assert Organization.new(:name => 'name', :label => 'with_underscore').valid?
+      assert Organization.new(:name => 'name', :label => 'With_Capital_letter').valid?
+      assert Organization.new(:name => 'name', :label => 'with_number').valid?
+      refute Organization.new(:name => 'name', :label => 'with\'space').valid?
```

Comment:
```
irb(main):001:0> 'with\'space'
=> "with'space"
```
seems like this should be called `'with\'apostrophe'` ?

Reply:
in my defense I copy/pasted these tests :)